charts/sn-platform/templates/openshift/_openshift.tpl

```diff
@@ -1,3 +1,12 @@
-{{- define "pulsar.openshift.scc.name" -}}
-{{ template "pulsar.namespace" . }}-{{ template "pulsar.fullname" . }}-openshift-scc
+{{- define "pulsar.openshift.scc" -}}
```

Comment:
Do we need to use `sccName`, `roleName`, and `rolebindingName` to make the vars' name clear?

Reply:
Just keep consistent with similar configs. e.g., https://github.com/streamnative/charts/blob/722bffe84e208e5f15f968520da6dbad5d0c34dd/charts/sn-platform/templates/zookeeper/_zookeeper.tpl#L272